crates/bevy_pbr/src/render/pbr_types.wgsl

```diff
@@ -44,6 +53,12 @@ fn standard_material_new() -> StandardMaterial {
     material.perceptual_roughness = 0.5;
     material.metallic = 0.00;
     material.reflectance = 0.5;
+    material.diffuse_transmission = 0.0;
+    material.transmission = 0.0;
+    material.thickness = 0.0;
+    material.ior = 1.5;
+    material.attenuation_distance = 1.0 / 0.0; // f32::INFINITY
```

Comment:
Does this work portably? Seems suspicious. I don't think the WGSL spec guarantees this will work...

Reply:
Oh no, looks like it doesn't. 😕  In fact, I suspect that if/when the compiler is fully compliant it will need to reject this as an error: 

<img width="816" alt="image" src="https://github.com/bevyengine/bevy/assets/418473/e424d08c-9e1c-443c-affd-ac9bca071c5c">

Reply:
Yeah a flag set on the CPU is probably the best way to do it.